src/components/JiscBoombox/JiscBoombox.stories.js

```diff
@@ -14,32 +14,38 @@ export default {
 export const Boombox1 = () => (
     <JiscBoombox backgroundImage='https://www.jisc.ac.uk/sites/default/files/abstract-blue-background-dark.jpg'>
         <Typography variant='h2'>Jisc Boombox Example 1</Typography>
+        <br />
         <Typography>
             Join our three-day online event and explore how to elevate the student experience during this challenging
             time
         </Typography>
+        <br />
         <JiscButton>Click me</JiscButton>
     </JiscBoombox>
 );
 
 export const Boombox2 = () => (
     <JiscBoombox backgroundImage='https://www.jisc.ac.uk/sites/default/files/abstract-blue-background-dark.jpg'>
         <Typography variant='h2'>Jisc Boombox Example 2</Typography>
+        <br />
         <Typography>
             Join our three-day online event and explore how to elevate the student experience during this challenging
             time
         </Typography>
+        <br />
         <JiscButton style={{ marginTop: jiscTheme.spacing(3) }}>Click Here</JiscButton>
```

Comment:
Shall we delete this hard-coded spacing, or do we want to leave it in as a different example?